packages/office-ui-fabric-react/src/components/Pivot/Pivot.tsx

```diff
@@ -1,291 +1,18 @@
-import * as React from 'react';
+import { styled } from '../../Utilities';
 import {
-  BaseComponent,
-  KeyCodes,
-  css,
-  getId,
-  createRef
-} from '../../Utilities';
-import { CommandButton } from '../../Button';
-import { IPivotProps } from './Pivot.types';
-import { IPivotItemProps } from './PivotItem.types';
-import { FocusZone, FocusZoneDirection } from '../../FocusZone';
-import { PivotItem } from './PivotItem';
-import { PivotLinkFormat } from './Pivot.types';
-import { PivotLinkSize } from './Pivot.types';
-import { Icon } from '../../Icon';
-import * as stylesImport from './Pivot.scss';
-const styles: any = stylesImport;
+  IPivotProps,
+  IPivotStyleProps,
+  IPivotStyles
+} from './Pivot.types';
+import { PivotBase } from './Pivot.base';
+import { getStyles } from './Pivot.styles';
 
 /**
- *  Usage:
- *
- *   <Pivot>
- *     <PivotItem linkText="Foo">
- *       <Label>Pivot #1</Label>
- *     </PivotItem>
- *     <PivotItem linkText="Bar">
- *       <Label>Pivot #2</Label>
- *     </PivotItem>
- *     <PivotItem linkText="Bas">
- *       <Label>Pivot #3</Label>
- *     </PivotItem>
- *   </Pivot>
+ * The Pivot control and related tabs pattern are used for navigating frequently accessed,
```

Comment:
Why move the "usage" to the base file but leave the "description" here?

Reply:
I just renamed the file and made the new styled file.